### ISSUE TITLE
[website docs] Package name for FreeBSD changed

### ISSUE THREAD

**Issue by AnotherKamila** [asciinema/asciinema#327]
TL;DR: https://asciinema.org/docs/installation#installing-on-freebsd should read `cd /usr/ports/textproc/py-asciinema...` and `pkg install py36-asciinema`.

Installation docs on the website (https://asciinema.org/docs/installation#installing-on-freebsd) say that the package name is `asciinema`, but this is no longer true, because FreeBSD changed how Python-based ports/packages work in order to easily support different versions of Python. (See [Flavors in the FreeBSD Porter's Handbook](https://www.freebsd.org/doc/en_US.ISO8859-1/books/porters-handbook/flavors.html) for a detailed explanation of what's happening here.)

The current naming of things is:

* the port is called `py-asciinema` and the user can configure which Python it will be built with -- so the ports installation option should be changed to `cd /usr/ports/textproc/py-asciinema && make install`
* the package is called `py<SOMETHING>-asciinema`, where `<SOMETHING>` is the version of Python that the pre-built package has been built with; this is currently Python 3.6, so the name of the current official package is `py36-asciinema`.

I would send a PR, but it seems that the website is not on GitHub, so I am filing the issue here.
